gibberish_server/models/room.js

```diff
@@ -71,6 +71,25 @@ class Room {
         }
     }
 
+    updatePlayers(nickname) {
+        var index = -1
```

Comment:
not sure if i understood this completely -> basically player last polled difference is greater than 1sec, they are taken out of the player list. But what if the player just stops polling get requests completely, how would the server check?

Reply:
every player in the room will poll the /room api every 500ms. during which, it will update their lastPolled value of the player that polled to the current timestamp. it will also check each player is the list whether their lastPolled value is more than 1 second away. if it is, then that player gets removed. however, if the last player of that room disconnects and hence no player calls the /room api for that room, then the check will stop, meaning that the database will be left with rooms with one player inside. im not sure how we can clean up the database for these empty rooms, but i think perhaps we can leave it as that for now?

Reply:
ok sure, guess this works! At least for the front-end there should be no issues. Yeah can probably try to remove empty rooms for the backend side next time.